src/main/groovy/org/checkerframework/gradle/plugin/CheckerFrameworkPlugin.groovy

```diff
@@ -42,6 +45,50 @@ final class CheckerFrameworkPlugin implements Plugin<Project> {
         if (!applied) applied = true
       }
     }
+
+    project.getPlugins().withType(io.freefair.gradle.plugins.lombok.LombokPlugin.class, new Action<LombokPlugin>() {
+      void execute(LombokPlugin lombokPlugin) {
+        project.gradle.projectsEvaluated {
+
+          def delombokTasks = project.getTasks().findAll { task ->
+            task.name.startsWith("delombok")
+          }
+
+          if (delombokTasks.size() != 0) {
+
+            // change every compile task so that it:
+            // 1. depends on delombok, and
+            // 2. uses the delombok'd source code as its source set
+
+            project.tasks.withType(AbstractCompile).all { compile ->
+
+              // find the right delombok task
+              def delombokTask = delombokTasks.find { task ->
+                if (task.name.equals("delombok")) {
+                  // special-case the main compile task because its just named "compileJava"
+                  // without anything else
+                  compile.name.equals("compileJava")
+                } else {
+                  // "delombok" is 8 characters.
+                  compile.name.contains(task.name.substring(8))
```

Comment:
I wonder if this should test equality rather than substring, because some tasks might be substrings of other tasks.  It's probably fine to leave it as is for now, though.

Reply:
I think it's fine as-is, but it is worth noting that we *must* check substring, because the compile task's name includes other information. For example, the standard test task is "compileTestJava", and the corresponding delombok task is "delombokTest".